controllers/usersController.js

```diff
@@ -0,0 +1,46 @@
+const {User, validate} = require('../models/user');
+
+exports.getAllUsers = async (req, res) => {
```

Comment:
instead of prefixing each handler with `exports`, you can just create them as normal functions:
`const getAllUsers = async (req, res) => { ..... }`

And export at the end of the usersController file like this:

`exports = { getAllUsers, ..., ..., }` 
OR
`module.exports = { getAllUsers, ..., ... }`